src/app/pages/ProposalDetailsPage/ProposalVotesCard.tsx

```diff
@@ -140,9 +139,9 @@ export const ProposalVotesCard: FC = () => {
         </div>
       </div>
       <ErrorBoundary light={true}>
-        <Box sx={{ height: '704px' }}>
+        <div className="h-[704px]">
```

Comment:
can we remove this hardcoded height? does not make sense to limit table wrapper this way